debian/control

```diff
@@ -4,26 +4,21 @@ Priority: optional
 Maintainer: Colin Curtain, <ccbogel@hotmail.com>
 Build-Depends: 
     debhelper (>= 7),
-    dh-python,
+    dh-virtualenv,
     python3-all,
+    python3-dev,
     python3-setuptools,
 Standards-Version: 3.9.2
 
 Package: qualcoder
 Architecture: all
+Pre-Depends: dpkg (>= 1.16.1), ${misc:Pre-Depends}
 Depends: 
-    python3, 
+    ${misc:Depends}
     python3-pyqt5,
     python3-pil, 
     python3-lxml, 
-    python3-pikepdf, 
     python3-six,
-    python3-ebooklib,
```

Comment:
why are ebooklib and others removed from here?

Reply:
because there are no packages python3-ebooklib in ubuntu bionic, for disco their might be.